apps/aechannel/src/aesc_fsm.erl

```diff
@@ -194,7 +207,7 @@ patterns() ->
                             not lists:member(F, [module_info, patterns])].
 
 record_fields(data ) -> record_info(fields, data);
-record_fields(w    ) -> record_info(fields, w);
+record_fields(w    ) -> aesc_window:record_fields(w);
 record_fields(Other) -> aesc_offchain_state:record_fields(Other).
 %% ==================================================================
 
```

Comment:
Is the state machine at line 222 up-to-date? Shall the copy of the FSM in the protocol repo be made canonical, this deleted, and[ the FSM in the protocol repo enhanced](https://github.com/aeternity/protocol/tree/master/channels#overview)?